### PR TITLE
Adds Chinese translation of v1.0.0-rc.2

### PR DESCRIPTION
**Pull request by aaranxu**
Hi all,

I added a Chinese version of the the TOML v1.0.0-rc.2 , based on the current latest one (cn/1.0.0-rc.1). 

I have seen some discussions on the review process of translation. The previous Chinese versions were written by @LongTengDao, @chenrui333 and @ZhangHanDong, and could you review my pull request, thanks so much.

I have also translated some documents of the Rust language, such as [**Rust By Example**](https://github.com/rust-lang-cn/rust-by-example-cn). For the Chinese translation and review of TOML project, maybe I can participate in the future, thank you.

### CODE REVIEW

**Comment by ZhangHanDong** [comment]
@aaranxu 辛苦，大致看了下，没有啥问题。

File: specs/cn/v1.0.0-rc.2.md [hunk 1]
```diff
@@ -0,0 +1,915 @@
...
+\f         - form feed       (U+000C)
+\r         - carriage return (U+000D)
+\"         - quote           (U+0022)
+\\         - backslash       (U+005C)
+\uXXXX     - unicode         (U+XXXX)
+\UXXXXXXXX - unicode         (U+XXXXXXXX)
+```
+
+任何 Unicode 字符都可以用 `\uXXXX` 或 `\UXXXXXXXX` 的形式来转义。  
+转义码必须是有效的 Unicode [标量值](http://unicode.org/glossary/#unicode_scalar_value)。
```
Comment (chenrui333) [inline specs/cn/v1.0.0-rc.2.md:263]:
```suggestion
转义码必须是有效的 Unicode [标量值](https://unicode.org/glossary/#unicode_scalar_value)。  
```

File: specs/cn/v1.0.0-rc.2.md [hunk 2]
```diff
@@ -0,0 +1,915 @@
...
+紧随开头引号的那个换行会被去除。  
+其它空白和换行会被原样保留。  
+
+```toml
+str1 = """
+Roses are red
+Violets are blue"""
+```
+
+TOML 解析器可以相对灵活地解析成对所在平台有效的换行字符。
```
Comment (chenrui333) [inline specs/cn/v1.0.0-rc.2.md:280]: 解释器 应该是比较标准的说法？
```suggestion
TOML 解释器可以相对灵活地解析成对所在平台有效的换行字符。  
```

File: specs/cn/v1.0.0-rc.2.md [hunk 3]
```diff
@@ -0,0 +1,915 @@
...
+Violets are blue"""
+```
+
+TOML 解析器可以相对灵活地解析成对所在平台有效的换行字符。  
+
+```toml
+# 在 Unix 系统，上面的多行字符串可能等同于：
+str2 = "Roses are red\nViolets are blue"
+
+# 在 Windows 系统，它可能等价于：
```
Comment (chenrui333) [inline specs/cn/v1.0.0-rc.2.md:286]:
```suggestion
# 在 Windows 系统上，它很可能等同于：
```

File: specs/cn/v1.0.0-rc.2.md (hunk 1 above)
Comment (chenrui333) [reply specs/cn/v1.0.0-rc.2.md:263]: realized this is the upstream issue, submit over [there](https://github.com/toml-lang/toml/pull/786).

**Review by chenrui333 — APPROVED** [review]
lgtm, 🚢

**Comment by ZhangHanDong** [comment]
lgtm，辛苦了

**Comment by pradyunsg** [comment]
Thanks for this @aaranxu and for the reviews @chenrui333 @ZhangHanDong! Much appreciated! ^>^

**Pull request merged**